src/normalize/normalize.py

```diff
@@ -110,17 +110,19 @@ def parse_files(s3, s3_fs, source_prefix, destination_prefix, start_date, state_
             str(cur_processing.day),
         )
         LOGGER.info(f"Processing date: {date_partition}")
-        max_timestamp = cur_processing.timestamp()
-
-        trip_updates_pa, vehicles_pa, alerts_pa = None, None, None
+        max_epoch_timestamp = cur_processing.timestamp()
 
         for page in paginator.paginate(Bucket=source_bucket, Prefix=date_partition, PaginationConfig={'PageSize': 60}):
+            trip_updates_pa = None
+            vehicles_pa = None
+            alerts_pa = None
```

Comment:
this was effectively the root cause, since we're writing a file for each pagination: we need to reset our state when we scan a new pagination result